Makefile

```diff
@@ -1,4 +1,4 @@
-serve:
+serve: get-milestones
```

Comment:
IMO we don't want to run this target before serve, because it'll require everyone to have a GitHub API token that can pull repo status.

Reply:
You only need the `npm install` from the _get-milestones_ target. Separating that from the `node -r esm ...` should be sufficient and won't requite GH tokens be configured.

Reply:
Done